t/live/repos.t

```diff
@@ -467,6 +467,39 @@ SKIP: {
             'Pithub::Repos::Hooks->hook_id not successful after delete';
 
     }
+    {
+        # Pithub::Repos::Releases->create
+        # Using JSON true/false values
+        require JSON::MaybeXS;
+        my $result = $p->repos->releases->create(
+            data => {
+                tag_name         => "v0.01000",
+                target_commitish => "master",
+                name             => "This is a test release of v0.01000",
+                body             => "Here be notes",
+                draft            => JSON::MaybeXS::false,
+                prerelease       => JSON::MaybeXS::false,
```

Comment:
Changing these to `JSON::MaybeXS::false()` may fix the error.

Reply:
@oalders done lets try again.